packages/rss-handler/src/index.ts

```diff
@@ -53,6 +58,30 @@ export const isOldItem = (item: RssFeedItem, lastFetchedAt: number) => {
   )
 }
 
+const feedFetchFailedRedisKey = (feedUrl: string) =>
+  `feed-fetch-failure:${feedUrl}`
+
+const isFeedBlocked = async (feedUrl: string, redisClient: RedisClient) => {
```

Comment:
awesome

Reply:
we probably do need a way of logging when this happens. Maybe just a log line is good enough for now

Reply:
Ideally would be a posthog event but we don't have that all setup yet

Reply:
noted